check-branch-name/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Check if the branch name follows our naming convention."
+description: "Action that checks if the name of the branch is following the our branch naming convention."
+
+runs:
+  using: composite
+  steps:
+    - uses: jungwinter/split@master
+      id: split
+      with:
+        msg: ${{ env.BRANCH_NAME }}
```

Comment:
As far as I know, there is no `env.BRANCH_NAME` in Github actions, see the [GitHub actions context](https://docs.github.com/en/actions/learn-github-actions/contexts).

There is a variable named `github.ref_name`, that refers to the branch or tag that triggered the workflow. If this variable is used with `github.ref_type` (possible values are "branch" or "tag"), the branch name can be extracted.

```yml
if: github.ref_type == 'branch':
  ...
```

Maybe, we could delegate this conditional to the user. What do you think?

Reply:
I don't see why we would want to delegate this conditional to the user. We always want to use this action on branches, not on tags, right? We can't perform any check on the tags with this action.